docs/3_2_model_exchange_api.adoc

```diff
@@ -190,7 +190,8 @@ Example: +
 Every implementation of the FMI must support calling sequences of the functions according to the following state chart:
 
 .Calling sequence of Model Exchange C functions in form of an UML 2.0 state machine.
-image::images/CallingSequenceME.png[width=100%]
+[#figure-model-exchange-state-machine]
+image::images/calling-sequence-model-exchange.svg[width=80%, align="center"]
 
 The objective of the start chart is to define the allowed calling sequences for functions of the FMI: Calling sequences not accepted by the state chart are not supported by the FMI.
 The behavior of an FMU is undefined for such a calling sequence.
```

Comment:
The description of the calling sequence for Model Exchange should be improved in the future and a common way of describing the allowed and disallowed function calls for ModelExchange and Co-Simulation sould be found.